contracts/deploy-erc20-fa12.js

```diff
@@ -166,32 +166,32 @@ const deployEthereumContract = async (ethConfig, argument, web3) => {
 const deploy = async (store, web3) => {
   try {
     console.log("\nContract Addresses :");
-    const tezosSwapContract = await deployTezosContract(
-      tezConfig,
-      `(Pair (Pair True "${store.keyStore.publicKeyHash}") (Pair "${config.tezos.tokenContract.address}" (Pair 15 {})))`,
-      store
-    );
-    console.log(`- Tezos Swap contract at ${tezosSwapContract}`);
+    // const tezosSwapContract = await deployTezosContract(
```

Comment:
if this isn't needed anymore, please delete it. if it's needed under specific conditions, wrap it in an `if`